.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+select = ANN,A50,B,B9,BLK,C,E,F,I,S,W,N
```

Comment:
Eg vil gjerne ha med D, for flake8-docstrings, som sikrer at alle public metoder har dokumentasjon.